flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveReader.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.hive.read;
+
+import org.apache.flink.table.dataformat.BaseRow;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+/**
+ * Hive reader to read record from files. The reader is only responsible for reading the data
+ * of a single split.
+ */
+public interface HiveReader extends Closeable {
```

Comment:
Is this interface specific to `Hive`?  if not, I would rename to something more generic, like `SplitReader`

Reply:
Good suggestion!